ui/pages/onboarding-flow/import-srp/import-srp.js

```diff
@@ -53,7 +56,7 @@ export default function ImportSRP({ submitSecretRecoveryPhrase }) {
             srpText={t('typeYourSRP')}
           />
           <Button
-            className="import-srp__confirm-button"
+            className="import-srp__actions__button"
```

Comment:
Updating to adhere to BEM naming convention

Reply:
Agreed, this violates our naming conventions